### ISSUE TITLE
Groups do not export on the CLI when collapsed

### ISSUE THREAD

**Issue by sanbox-irl** [aseprite/aseprite#2484]
Hello! I'm a programmer on a game called Fields of Mistria, and we use the aseprite cli pretty extensively in our tooling. We're running into an odd problem though.

When running certain commands using the CLI on Aseprite files with folders, folders which are collapsed are not exported, even if they are visible and even if "all-layers" is passed. Is this a bug or expected behavior? If this is expected, is there a way to override folders via the cli to be open? Is there a way to work around this?

## Steps to Repro:
1. Download this example aseprite file: 
[Group Issue Example.aseprite.zip](https://github.com/aseprite/aseprite/files/5060069/Group.Issue.Example.aseprite.zip)

2. Open the aseprite file and confirm that the Group "Group 1" is collapse (its parent folder, "Group 2" should be uncollapsed). If it's not that way, make it that way.

3. run the following command (i did this in zsh but it shouldn't matter ofc):
```sh
aseprite="path/to/aseprite"

$aseprite -b --split-layers --filename-format {frame}-{layer} "Group Issue Example.aseprite" --sheet "Group Issue Example.png" --format json-array --data "Group Issue Example.json" --list-tags --list-layers --all-layers
```

(this is the actual command that we run in Fields of Mistria's tooling -- I haven't extensively tested other configurations)

4. Observe that `Group Issue Example.json` only lists the following layers:
```json
  "layers": [
   { "name": "Group 2" },
   { "name": "layer 0", "group": "Group 2", "opacity": 255, "blendMode": "normal" }
  ]
```

As well, notice that the `png` is not malformed according to this json -- ie, it is only showing the layers the json manifest is describing. That's good, but also bad, because it means that we can't use other methods (such as parsing the frame strings) to figure out the layer data, since the layers just weren't outputted at all.

## Tl;dr
If you import any groups with a collapsed folder:

![image](https://user-images.githubusercontent.com/31973761/89963058-6a9a6900-dbfb-11ea-8a2b-8c22988fd34b.png)

those layers and frames within that collapsed folders will not be exported when using the CLI.

This behavior is *not* reflected on using the "export sprite sheet" in the ui, which will not care that the folders are collapsed.

Thank you so much for taking to time to read all of this! It is very appreciated

### Aseprite and System version
* Aseprite version: 1.2.23
System: Windows/macOS, distro via Steam and Humble (tested on all)

**Comment by dacap** [comment]
Hi @sanbox-irl! I was able to reproduce the bug, one possible workaround at the moment could be this:

```
$aseprite -b \
  --split-layers \
  --filename-format {frame}-{layer} \
  "Group Issue Example.aseprite" \
  --script expand_all.lua \
  --sheet "Group Issue Example.png" \
  --format json-array \
  --data "Group Issue Example.json" \
  --list-tags --list-layers --all-layers
```

The key is the new `--script expand_all.lua` argument after loading the .aseprite file, where `expand_all.lua` is a little script to expand all groups:

```lua
local s = app.activeSprite
local function expandLayers(l)
  if l.isGroup then
    l.isExpanded = true
    for _,c in ipairs(l.layers) do
      expandLayers(c)
    end
  end
end
for _,l in ipairs(s.layers) do
  expandLayers(l)
end
```

We hope it is a good workaround until the final fix is available 🤞

**Comment by sanbox-irl** [comment]
looks great :) thank you for the quick response -- I really appreciate it

**Comment by sanbox-irl** [comment]
nice stuff! and we made the release too! thank you very very much
